src/redux/features/taskBoardSlice.tsx

```diff
@@ -56,6 +56,13 @@ const taskBoardSlice = createSlice({
           : state.tasks.filter((task) => !task.assigneeId)
       state.filteredTasks = filteredTasks
     },
+    setFilteredTaskByType: (state, action) => {
+      const assigneeType: string[] = action.payload?.assigneeType
```

Comment:
The type for assignee type should be made stricter using `AssigneeType` from `@prisma/client`

Reply:
`(AssigneeType | 'all')[]` in your case *